### ISSUE TITLE
zed use 100% CPU if you use multipath-policy multibus

### ISSUE THREAD

**Issue by colttt** [openzfs/zfs#8339]
### System information
<!--  add version after "|" character -->
Type | Version/Name
 --- | --- 
Distribution Name	|  Debian GNU/Linux 9.6 (stretch)
Distribution Version	|  9.6
Linux Kernel	|  Linux tani 4.19.0-0.bpo.1-amd64 #1 SMP Debian 4.19.12-1~bpo9+1 (2018-12-30) x86_64 GNU/Linux
Architecture	|  x86_64
ZFS Version	|  0.7.12-1~bpo9+1
SPL Version	|  0.7.12-1~bpo9+1

### Describe the problem you're observing
If I use the multipath-policy "multibus" zed use 100% CPU, If a use the default policy (failover), everything looks normal.

### Describe how to reproduce the problem
install multipath and and set the following config:
```
/etc/multipath.conf 
defaults {
        path_grouping_policy    multibus
}
```
after a reboot it looks like that:
```
35000c500a721847f dm-118 SEAGATE,ST10000NM0096
size=9.1T features='1 retain_attached_hw_handler' hwhandler='0' wp=rw
`-+- policy='service-time 0' prio=1 status=active
  |- 15:0:67:0  sdbs 68:96   active ready running
  `- 16:0:67:0  sdgi 131:224 active ready running
35000c500a7222767 dm-99 SEAGATE,ST10000NM0096
size=9.1T features='1 retain_attached_hw_handler' hwhandler='0' wp=rw
`-+- policy='service-time 0' prio=1 status=active
  |- 15:0:95:0  sdct 70:16   active ready running
  `- 16:0:95:0  sdhj 133:144 active ready running
35000c500a720be27 dm-78 SEAGATE,ST10000NM0096
size=9.1T features='1 retain_attached_hw_handler' hwhandler='0' wp=rw
`-+- policy='service-time 0' prio=1 status=active
  |- 15:0:84:0  sdcj 69:112  active ready running
  `- 16:0:84:0  sdgz 132:240 active ready running
```
(I've 120 disks in 2JBODs which are connected via 2HBAs)

and you will have 100% CPU
```
top - 11:55:57 up  1:22,  4 users,  load average: 1.12, 1.13, 1.71
Tasks: 694 total,   2 running, 407 sleeping,   0 stopped,   0 zombie
%Cpu(s):  1.3 us,  2.9 sy,  0.0 ni, 95.7 id,  0.0 wa,  0.0 hi,  0.1 si,  0.0 st
GiB Mem :  251.584 total,  249.651 free,    1.642 used,    0.290 buff/cache
GiB Swap:    7.449 total,    7.449 free,    0.000 used.  248.229 avail Mem 

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND                                                                                                                                                                                                                                                  
 3706 root      20   0   33544   1692   1464 R 100.0  0.0  80:54.34 zed                                                                                                                                                                                                                                                      
 1365 root      rt   0 8402988  36012   6828 S   0.7  0.0   0:21.07 multipathd 
```
### Include any warning/errors/backtraces from the system logs
I don't see very anything in the logs. If I use strace I got the following:
```
strace -ir -o zed.out -p 3706
     0.000000 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000123 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000059 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000052 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000053 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
[...]
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000026 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
     0.000025 [00007f327d62add7] ioctl(5, _IOC(0, 0x5a, 0x81, 0x00), 0x7fff13b64f90) = -1 EBADF (Bad file descriptor)
```
a lot of lines:
```
wc -l zed.out
8165690 zed.out
```

Edit:
```
strace -itt -o zed.strace.log zed -vF
output:
4:25:30.717358 [00007fcc13e6f647] execve("/usr/sbin/zed", ["zed", "-vF"], [/* 18 vars */]) = 0
14:25:30.717997 [00007f9aa7bb05a9] brk(NULL) = 0x55b820a6f000
14:25:30.718103 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.718187 [00007f9aa7bb13ca] mmap(NULL, 12288, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f9aa7db8000
14:25:30.718266 [00007f9aa7bb12d7] access("/etc/ld.so.preload", R_OK) = -1 ENOENT (No such file or directory)
14:25:30.718343 [00007f9aa7bb1277] open("/etc/ld.so.cache", O_RDONLY|O_CLOEXEC) = 3
14:25:30.718420 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=27762, ...}) = 0
14:25:30.718496 [00007f9aa7bb13ca] mmap(NULL, 27762, PROT_READ, MAP_PRIVATE, 3, 0) = 0x7f9aa7db1000
14:25:30.718566 [00007f9aa7bb1377] close(3) = 0
14:25:30.718636 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.718715 [00007f9aa7bb1277] open("/lib/libnvpair.so.1", O_RDONLY|O_CLOEXEC) = 3
14:25:30.718792 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0`I\0\0\0\0\0\0"..., 832) = 832
14:25:30.718866 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=79856, ...}) = 0
14:25:30.718938 [00007f9aa7bb13ca] mmap(NULL, 2175056, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa7984000
14:25:30.719007 [00007f9aa7bb1467] mprotect(0x7f9aa7997000, 2093056, PROT_NONE) = 0
14:25:30.719083 [00007f9aa7bb13ca] mmap(0x7f9aa7b96000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x12000) = 0x7f9aa7b96000
14:25:30.719175 [00007f9aa7bb1377] close(3) = 0
14:25:30.719249 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.719323 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/librt.so.1", O_RDONLY|O_CLOEXEC) = 3
14:25:30.719414 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\340 \0\0\0\0\0\0"..., 832) = 832
14:25:30.719488 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=31744, ...}) = 0
14:25:30.719560 [00007f9aa7bb13ca] mmap(NULL, 2128832, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa777c000
14:25:30.719630 [00007f9aa7bb1467] mprotect(0x7f9aa7783000, 2093056, PROT_NONE) = 0
14:25:30.719704 [00007f9aa7bb13ca] mmap(0x7f9aa7982000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x6000) = 0x7f9aa7982000
14:25:30.719795 [00007f9aa7bb1377] close(3) = 0
14:25:30.719866 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.719940 [00007f9aa7bb1277] open("/lib/libzpool.so.2", O_RDONLY|O_CLOEXEC) = 3
14:25:30.720013 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\260U\2\0\0\0\0\0"..., 832) = 832
14:25:30.720085 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=1270808, ...}) = 0
14:25:30.720161 [00007f9aa7bb13ca] mmap(NULL, 4828536, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa72e1000
14:25:30.720230 [00007f9aa7bb1467] mprotect(0x7f9aa7410000, 2093056, PROT_NONE) = 0
14:25:30.720299 [00007f9aa7bb13ca] mmap(0x7f9aa760f000, 32768, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x12e000) = 0x7f9aa760f000
14:25:30.720378 [00007f9aa7bb13ca] mmap(0x7f9aa7617000, 1461624, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x7f9aa7617000
14:25:30.720463 [00007f9aa7bb1377] close(3) = 0
14:25:30.720545 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.720617 [00007f9aa7bb1277] open("/lib/libzfs.so.2", O_RDONLY|O_CLOEXEC) = 3
14:25:30.720692 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0@\235\0\0\0\0\0\0"..., 832) = 832
14:25:30.720763 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=276592, ...}) = 0
14:25:30.720835 [00007f9aa7bb13ca] mmap(NULL, 2371944, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa709d000
14:25:30.720902 [00007f9aa7bb1467] mprotect(0x7f9aa70df000, 2093056, PROT_NONE) = 0
14:25:30.720972 [00007f9aa7bb13ca] mmap(0x7f9aa72de000, 12288, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x41000) = 0x7f9aa72de000
14:25:30.721083 [00007f9aa7bb1377] close(3) = 0
14:25:30.721159 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.721233 [00007f9aa7bb1277] open("/lib/libzfs_core.so.1", O_RDONLY|O_CLOEXEC) = 3
14:25:30.721308 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\0\21\0\0\0\0\0\0"..., 832) = 832
14:25:30.721379 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=14176, ...}) = 0
14:25:30.721451 [00007f9aa7bb13ca] mmap(NULL, 2109584, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa6e99000
14:25:30.721519 [00007f9aa7bb1467] mprotect(0x7f9aa6e9c000, 2093056, PROT_NONE) = 0
14:25:30.721589 [00007f9aa7bb13ca] mmap(0x7f9aa709b000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x2000) = 0x7f9aa709b000
14:25:30.721677 [00007f9aa7bb1377] close(3) = 0
14:25:30.721750 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.721822 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/libuuid.so.1", O_RDONLY|O_CLOEXEC) = 3
14:25:30.721898 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0p\25\0\0\0\0\0\0"..., 832) = 832
14:25:30.721970 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=19008, ...}) = 0
14:25:30.722041 [00007f9aa7bb13ca] mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f9aa7daf000
14:25:30.722121 [00007f9aa7bb13ca] mmap(NULL, 2113936, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa6c94000
14:25:30.722190 [00007f9aa7bb1467] mprotect(0x7f9aa6c98000, 2093056, PROT_NONE) = 0
14:25:30.722259 [00007f9aa7bb13ca] mmap(0x7f9aa6e97000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x3000) = 0x7f9aa6e97000
14:25:30.722346 [00007f9aa7bb1377] close(3) = 0
14:25:30.722418 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.722491 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/libz.so.1", O_RDONLY|O_CLOEXEC) = 3
14:25:30.722565 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\300!\0\0\0\0\0\0"..., 832) = 832
14:25:30.722637 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=105088, ...}) = 0
14:25:30.722709 [00007f9aa7bb13ca] mmap(NULL, 2200072, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa6a7a000
14:25:30.722779 [00007f9aa7bb1467] mprotect(0x7f9aa6a93000, 2093056, PROT_NONE) = 0
14:25:30.722850 [00007f9aa7bb13ca] mmap(0x7f9aa6c92000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x18000) = 0x7f9aa6c92000
14:25:30.722936 [00007f9aa7bb1377] close(3) = 0
14:25:30.723012 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.723085 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/libpthread.so.0", O_RDONLY|O_CLOEXEC) = 3
14:25:30.723159 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0Pa\0\0\0\0\0\0"..., 832) = 832
14:25:30.723231 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0755, st_size=135440, ...}) = 0
14:25:30.723303 [00007f9aa7bb13ca] mmap(NULL, 2212936, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa685d000
14:25:30.723372 [00007f9aa7bb1467] mprotect(0x7f9aa6875000, 2093056, PROT_NONE) = 0
14:25:30.723453 [00007f9aa7bb13ca] mmap(0x7f9aa6a74000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x17000) = 0x7f9aa6a74000
14:25:30.723533 [00007f9aa7bb13ca] mmap(0x7f9aa6a76000, 13384, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x7f9aa6a76000
14:25:30.723613 [00007f9aa7bb1377] close(3) = 0
14:25:30.723688 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.723761 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/libc.so.6", O_RDONLY|O_CLOEXEC) = 3
14:25:30.723835 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\3\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\0\4\2\0\0\0\0\0"..., 832) = 832
14:25:30.723924 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0755, st_size=1689360, ...}) = 0
14:25:30.724002 [00007f9aa7bb13ca] mmap(NULL, 3795296, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa64be000
14:25:30.724071 [00007f9aa7bb1467] mprotect(0x7f9aa6653000, 2097152, PROT_NONE) = 0
14:25:30.724143 [00007f9aa7bb13ca] mmap(0x7f9aa6853000, 24576, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x195000) = 0x7f9aa6853000
14:25:30.724221 [00007f9aa7bb13ca] mmap(0x7f9aa6859000, 14688, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x7f9aa6859000
14:25:30.724307 [00007f9aa7bb1377] close(3) = 0
14:25:30.724387 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.724460 [00007f9aa7bb1277] open("/lib/libuutil.so.1", O_RDONLY|O_CLOEXEC) = 3
14:25:30.724535 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0@S\0\0\0\0\0\0"..., 832) = 832
14:25:30.724607 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=72632, ...}) = 0
14:25:30.724678 [00007f9aa7bb13ca] mmap(NULL, 2172376, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa62ab000
14:25:30.724746 [00007f9aa7bb1467] mprotect(0x7f9aa62bc000, 2093056, PROT_NONE) = 0
14:25:30.724815 [00007f9aa7bb13ca] mmap(0x7f9aa64bb000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x10000) = 0x7f9aa64bb000
14:25:30.724893 [00007f9aa7bb13ca] mmap(0x7f9aa64bd000, 1496, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x7f9aa64bd000
14:25:30.724974 [00007f9aa7bb1377] close(3) = 0
14:25:30.725072 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.725145 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/libm.so.6", O_RDONLY|O_CLOEXEC) = 3
14:25:30.725220 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\3\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\200V\0\0\0\0\0\0"..., 832) = 832
14:25:30.725293 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=1063328, ...}) = 0
14:25:30.725365 [00007f9aa7bb13ca] mmap(NULL, 3158248, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa5fa7000
14:25:30.725435 [00007f9aa7bb1467] mprotect(0x7f9aa60aa000, 2093056, PROT_NONE) = 0
14:25:30.725493 [00007f9aa7bb13ca] mmap(0x7f9aa62a9000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x102000) = 0x7f9aa62a9000
14:25:30.725552 [00007f9aa7bb1377] close(3) = 0
14:25:30.725600 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.725642 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/libdl.so.2", O_RDONLY|O_CLOEXEC) = 3
14:25:30.725685 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\200\r\0\0\0\0\0\0"..., 832) = 832
14:25:30.725726 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=14640, ...}) = 0
14:25:30.725766 [00007f9aa7bb13ca] mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f9aa7dad000
14:25:30.725812 [00007f9aa7bb13ca] mmap(NULL, 2109680, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa5da3000
14:25:30.725853 [00007f9aa7bb1467] mprotect(0x7f9aa5da6000, 2093056, PROT_NONE) = 0
14:25:30.725899 [00007f9aa7bb13ca] mmap(0x7f9aa5fa5000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x2000) = 0x7f9aa5fa5000
14:25:30.725958 [00007f9aa7bb1377] close(3) = 0
14:25:30.726002 [00007f9aa7bb12d7] access("/etc/ld.so.nohwcap", F_OK) = -1 ENOENT (No such file or directory)
14:25:30.726044 [00007f9aa7bb1277] open("/lib/x86_64-linux-gnu/libblkid.so.1", O_RDONLY|O_CLOEXEC) = 3
14:25:30.726088 [00007f9aa7bb1297] read(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\340\217\0\0\0\0\0\0"..., 832) = 832
14:25:30.726129 [00007f9aa7bb1202] fstat(3, {st_mode=S_IFREG|0644, st_size=283464, ...}) = 0
14:25:30.726170 [00007f9aa7bb13ca] mmap(NULL, 2383392, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x7f9aa5b5d000
14:25:30.726211 [00007f9aa7bb1467] mprotect(0x7f9aa5b9e000, 2093056, PROT_NONE) = 0
14:25:30.726255 [00007f9aa7bb13ca] mmap(0x7f9aa5d9d000, 20480, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x40000) = 0x7f9aa5d9d000
14:25:30.726311 [00007f9aa7bb13ca] mmap(0x7f9aa5da2000, 3616, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x7f9aa5da2000
14:25:30.726363 [00007f9aa7bb1377] close(3) = 0
14:25:30.726510 [00007f9aa7bb13ca] mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f9aa7dab000
14:25:30.726555 [00007f9aa7bb13ca] mmap(NULL, 16384, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f9aa7da7000
14:25:30.726601 [00007f9aa7b98bb8] arch_prctl(ARCH_SET_FS, 0x7f9aa7da8780) = 0
14:25:30.726783 [00007f9aa7bb1467] mprotect(0x7f9aa6853000, 16384, PROT_READ) = 0
14:25:30.726834 [00007f9aa7bb1467] mprotect(0x7f9aa6e97000, 4096, PROT_READ) = 0
14:25:30.726903 [00007f9aa7bb1467] mprotect(0x7f9aa5d9d000, 16384, PROT_READ) = 0
14:25:30.726963 [00007f9aa7bb1467] mprotect(0x7f9aa5fa5000, 4096, PROT_READ) = 0
14:25:30.727039 [00007f9aa7bb1467] mprotect(0x7f9aa62a9000, 4096, PROT_READ) = 0
14:25:30.727102 [00007f9aa7bb1467] mprotect(0x7f9aa6a74000, 4096, PROT_READ) = 0
14:25:30.727157 [00007f9aa7bb1467] mprotect(0x7f9aa7982000, 4096, PROT_READ) = 0
14:25:30.727204 [00007f9aa7bb1467] mprotect(0x7f9aa6c92000, 4096, PROT_READ) = 0
14:25:30.727361 [00007f9aa7bb1467] mprotect(0x7f9aa64bb000, 4096, PROT_READ) = 0
14:25:30.727486 [00007f9aa7bb1467] mprotect(0x7f9aa7b96000, 4096, PROT_READ) = 0
14:25:30.727556 [00007f9aa7bb1467] mprotect(0x7f9aa709b000, 4096, PROT_READ) = 0
14:25:30.728353 [00007f9aa7bb1467] mprotect(0x7f9aa760f000, 20480, PROT_READ) = 0
14:25:30.728695 [00007f9aa7bb1467] mprotect(0x7f9aa72de000, 8192, PROT_READ) = 0
14:25:30.728826 [00007f9aa7bb1467] mprotect(0x55b8208e6000, 4096, PROT_READ) = 0
14:25:30.728879 [00007f9aa7bb1467] mprotect(0x7f9aa7dbb000, 4096, PROT_READ) = 0
14:25:30.728911 [00007f9aa7bb1447] munmap(0x7f9aa7db1000, 27762) = 0
14:25:30.728950 [00007f9aa6862dc2] set_tid_address(0x7f9aa7da8a50) = 906
14:25:30.728980 [00007f9aa6862e24] set_robust_list(0x7f9aa7da8a60, 24) = 0
14:25:30.729013 [00007f9aa686e1be] rt_sigaction(SIGRTMIN, {sa_handler=0x7f9aa6862bd0, sa_mask=[], sa_flags=SA_RESTORER|SA_SIGINFO, sa_restorer=0x7f9aa686e0c0}, NULL, 8) = 0
14:25:30.729051 [00007f9aa686e1be] rt_sigaction(SIGRT_1, {sa_handler=0x7f9aa6862c60, sa_mask=[], sa_flags=SA_RESTORER|SA_RESTART|SA_SIGINFO, sa_restorer=0x7f9aa686e0c0}, NULL, 8) = 0
14:25:30.729084 [00007f9aa6862f00] rt_sigprocmask(SIG_UNBLOCK, [RTMIN RT_1], NULL, 8) = 0
14:25:30.729123 [00007f9aa659e897] getrlimit(RLIMIT_STACK, {rlim_cur=8192*1024, rlim_max=RLIM64_INFINITY}) = 0
14:25:30.729234 [00007f9aa659eca9] brk(NULL) = 0x55b820a6f000
14:25:30.729264 [00007f9aa659eca9] brk(0x55b820a90000) = 0x55b820a90000
14:25:30.729315 [00007f9aa6576f47] geteuid() = 0
14:25:30.729345 [00007f9aa659e897] getrlimit(RLIMIT_NOFILE, {rlim_cur=1024, rlim_max=1024*1024}) = 0
14:25:30.729377 [00007f9aa686d250] close(3) = -1 EBADF (Bad file descriptor)
14:25:30.729407 [00007f9aa686d250] close(4) = -1 EBADF (Bad file descriptor)
14:25:30.729437 [00007f9aa686d250] close(5) = -1 EBADF (Bad file descriptor)
14:25:30.729467 [00007f9aa686d250] close(6) = -1 EBADF (Bad file descriptor)
14:25:30.729497 [00007f9aa686d250] close(7) = -1 EBADF (Bad file descriptor)
14:25:30.729526 [00007f9aa686d250] close(8) = -1 EBADF (Bad file descriptor)
14:25:30.729556 [00007f9aa686d250] close(9) = -1 EBADF (Bad file descriptor)
14:25:30.729586 [00007f9aa686d250] close(10) = -1 EBADF (Bad file descriptor)
14:25:30.729615 [00007f9aa686d250] close(11) = -1 EBADF (Bad file descriptor)
[...]
14:25:41.244491 [00007f9aa686d250] close(453836) = -1 EBADF (Bad file descriptor)
14:25:41.244515 [00007f9aa686d250] close(453837) = -1 EBADF (Bad file descriptor)
14:25:41.244538 [00007f9aa686d250] close(453838) = -1 EBADF (Bad file descriptor)
14:25:41.244565 [00007f9aa686d250] --- SIGINT {si_signo=SIGINT, si_code=SI_KERNEL} ---
14:25:41.244769 [????????????????] +++ killed by SIGINT +++
```
within ~10seconds its about 36M

**Comment by colttt** [comment]
hm does nobody use zfs in a multipath-enviroment and has this issue?

**Comment by colttt** [comment]
It looks like it is resolved with zfs 2.0.3, multipath-tools 0.7.9 and kernel 5.10
So I close this here
